python/lsst/daf/butler/core/dimensions/_dataCoordinateIterable.py

```diff
@@ -722,8 +722,8 @@ def __eq__(self, other: Any) -> bool:
     def __getitem__(self, index: int) -> DataCoordinate:
         pass
 
-    @overload  # noqa: F811
-    def __getitem__(self, index: slice) -> DataCoordinateSequence:
+    @overload  # noqa: F811 (FIXME: remove for py 3.8+)
+    def __getitem__(self, index: slice) -> DataCoordinateSequence:  # noqa F811
```

Comment:
Technically I think these all need to be `noqa: F811` -- without the colon it squashes all warnings for the line, with the colon I thought it only squashes F811.

Reply:
fixed this and other PRs too - I will do a run through jenkins before merging.